dcc_config.csv

```diff
@@ -2,9 +2,10 @@ project_name,synapse_asset_view,data_model_url,template_menu_config_file,logo_lo
 DCA Demo,syn33715412,https://raw.githubusercontent.com/Sage-Bionetworks/data-models/main/example.model.jsonld,demo/dca-template-config.json,demo/Logo_Sage_Logomark.png,https://synapse.org,google_sheet,FALSE,FALSE,TRUE,replace,table_and_file,FALSE,FALSE,#2a668d,#184e71,#191919
 Cancer Complexity Knowledge Portal - Database,syn27210848,https://raw.githubusercontent.com/mc2-center/data-models/main/mc2.model.jsonld,MC2/dca-template-config.json,MC2/cckp_logo.png,https://cancercomplexity.synapse.org,excel,FALSE,FALSE,FALSE,upsert,table_and_file,FALSE,FALSE,#407BA0,#5BB0B5,#191919
 AD Knowledge Portal,syn51324810,https://raw.githubusercontent.com/adknowledgeportal/data-models/main/divco.data.model.v1.jsonld,ADKP/dca-template-config.json,ADKP/ADKnowledgePortal.png,https://adknowledgeportal.synapse.org,excel,FALSE,FALSE,TRUE,replace,file_only,FALSE,FALSE,#2a668d,#184e71,#191919
-VEOIBD,syn51397378,https://raw.githubusercontent.com/VEOIBD/data_models/main/veoibd.data.model.jsonld,VEOIBD/dca-template-config.json,VEOIBD/VEOIBD Logo.png,https://veoibd.org,excel,FALSE,TRUE,TRUE,upsert,file_only,FALSE,FALSE,#2a668d,#184e72,#191920
+VEOIBD,syn51397378,https://raw.githubusercontent.com/VEOIBD/data_models/main/veoibd.data.model.jsonld,VEOIBD/dca-template-config.json,VEOIBD/VEOIBD Logo.png,https://veoibd.org,excel,FALSE,TRUE,TRUE,upsert,file_entities,FALSE,FALSE,#2a668d,#184e72,#191920

```

Comment:
I think this needs to be `file_and_entities` https://sage-schematic.readthedocs.io/en/develop/cli_reference.html#cmdoption-schematic-model-submit-mrt

Reply:
Corrected. Thanks for catching this.